requirements.txt

```diff
@@ -0,0 +1,7 @@
+click
+ply
+pytest
+pyyaml
+rdflib
+uritools
+xmltodict
```

Comment:
I would prefer to avoid duplication here as we already have the dependencies in the `pyproject.toml`.

Reply:
Good to know 😄